### PR TITLE
Issue 8514: Make paging for search items work again

### LINKED ISSUE #8514: Saved Searches - next page broken
### Expected behavior
When I click on a saved search (e.g. #art) in the left side widget I get a stream of items tagged with #art. When I want to get more items I click on the 'Next' button at the bottom of the page. The next page with items should show up.

### Actual behavior
The first page works. But when I click on 'Next' I get a page with no items. This happens with often used tags like #art or #newhere too. So, there should be content but it isn't displayed.

### Steps to reproduce the problem
Follow a hashtag by adding it to the saved searches. Click on that saved search. Try to go to page two.

### Friendica version you encountered the problem 
2020.03
2020.06-dev

### Friendica source (git, zip)
git

### PHP version
7.3

### SQL version
10.3 MariaDB

### PR DESCRIPTION
**Pull request by annando**
Fixes #8514

### CODE REVIEW

**Comment by annando** [comment]
@MrPetovan have you forgot about this one? Or is there some issue that you want to have changed?

**Comment by MrPetovan** [comment]
I had a review in the works but I didn't submit it.

File: src/Content/Text/HTML.php [hunk 1]
```diff
@@ -917,7 +918,7 @@ public static function search($s, $id = 'search-box', $aside = true)
 			'$save_label'   => $save_label,
 			'$search_hint'  => DI::l10n()->t('@name, !forum, #tags, content'),
 			'$mode'         => $mode,
-			'$return_url'   => urlencode('search?q=' . urlencode($s)),
+			'$return_url'   => urlencode(Search::getSearchPath($s)),
```
[the remark this answers is not in the document]
Comment (annando) [reply src/Content/Text/HTML.php:921]: I will check what the `return_url` is doing, but I guess it has got nothing to do with the pager.

[the remark this answers is not in the document]
Comment (annando) [reply src/Content/Text/HTML.php:921]: I just had a quick look. This is unrelated to the problem. This is just a variable for the  `searchbox.tpl` template. It isn't used in conjunction with the pager.

[the remark this answers is not in the document]
Comment (MrPetovan) [reply src/Content/Text/HTML.php:921]: It still looks wrong to me and it's going to bite us back again in the future, I'm predicting.

**Pull request merged**